packages/app/src/app/pages/Dashboard/Content/routes/Repositories/index.tsx

```diff
@@ -126,26 +69,16 @@ export const RepositoriesPage = () => {
       items={itemsToShow}
     >
       <Helmet>
-        <title>{path || 'Repositories'} - CodeSandbox</title>
+        <title>Repositories - CodeSandbox</title>
       </Helmet>
       <Header
         activeTeam={activeTeam}
-        path={path}
         showViewOptions
         showBetaBadge
-        nestedPageType={pageType}
-        selectedRepo={selectedRepo}
-        readOnly={isReadOnlyRepo}
+        title="All repositories"
       />
 
-      {isReadOnlyRepo && (
-        <Element paddingX={4} paddingY={2}>
-          {selectedRepo?.private && <PrivateRepoFreeTeam />}
-        </Element>
-      )}
-
-      {!selectedRepo &&
-      (hasMaxPublicRepositories || hasMaxPrivateRepositories) ? (
+      {hasMaxPublicRepositories || hasMaxPrivateRepositories ? (
```

Comment:
Maybe `hasMaxPublicRepositories || hasMaxPrivateRepositories` should just be `hasMaxRepos` since the check is done twice in this file.

Reply:
I saw this but I wouldn't introduce the change in this PR, as it's already grown a bit out of scope